arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.series

```diff
@@ -0,0 +1,31 @@
+# Kconfig.defconfig.series - ST Microelectronics STM32F7 MCU line
+#
+# Copyright (c) 2018 Yurii Hamann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_STM32F7X
+
+source "arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.stm32f7*"
```

Comment:
Replace source with gsource, this is causing CI to fail.
This is due to new commit on master

Reply:
I have fixed it and updated the commits.